ert/data/record/_transmitter.py

```diff
@@ -254,10 +110,12 @@ async def dump(self, location: Path, mime: str) -> None:
         record = await self.load()
         if isinstance(record, NumericalRecord):
             async with aiofiles.open(str(location), mode="wt", encoding="utf-8") as ft:
-                await ft.write(get_serializer(mime).encode(record.data))
-        else:
+                await ft.write(
+                    ert.serialization.get_serializer(mime).encode(record.data)
+                )
+        elif isinstance(record, BlobRecord):
```

Comment:
I think this is an improvement! But I would perhaps still like there to be an `else` raising an exception at the end to ensure that the record fall through the entire logic without any of the cases applying...